README.md

```diff
@@ -1,14 +1,35 @@
-### Hexlet tests and linter status:
+# Brain Games
+
+## Project description:
+#### A small pack of fun games for your terminal
+
+## Project installation:
+### Run commands from package root catalog:
+1. Poetry installation:
```

Comment:
it's better to use action form here, like `install project dependencies, build a package`